### DISCUSSION TITLE
Two Minor Changes Increased Net Profit %

### DISCUSSION THREAD

**Discussion by thouravi** [AlbertoCuadra/algo_trading_weighted_strategy#52] [Community settings]
Hey,

I tested your strategy and on changing two values, it increased 
- Net Profit from **1819%** to **2206%** 
- Profit Factor from **4.482** to **8.152**
on 4H BTC/USDT Binance.

**Change 1: Line 73**

Before - 27
`short_RSI = input.float(27, 'Exit SRSI Short (%)', minval=0.0, step=1, group='Strategy 2: Stoch RSI')`

My Change - 21
`short_RSI = input.float(21, 'Exit SRSI Short (%)', minval=0.0, step=1, group='Strategy 2: Stoch RSI')`

**Change 2: Line 92**

Before - 88
`MA2_period_5 = input.int(88, 'MA 2', minval=1, maxval=9999, step=1, group='Strategy 5: MA CROSS')`

My Change - 85
`MA2_period_5 = input.int(85, 'MA 2', minval=1, maxval=9999, step=1, group='Strategy 5: MA CROSS')`

**Change 3 - Line 42**

Before
`MAX_TP = input.int(6, 'Max number of TP', minval=1, maxval=6, group='Take Profits')`

My Change
`MAX_TP = input.int(3, 'Max number of TP', minval=1, maxval=6, group='Take Profits')`

**Change 4 - Line 36**

Before
`stoploss_perc= input.float(6.0, 'Stop Loss %', minval=0, maxval=100, group='Stop loss') * 0.01`

My Change
`stoploss_perc= input.float(8, 'Stop Loss %', minval=0, maxval=100, group='Stop loss') * 0.01`

Let me know what do you think, my username on TradingView is ImThour.
Let's have a chat, cheers.

**Comment by thouravi** [comment]
![image](https://user-images.githubusercontent.com/15676184/156870241-fa60c586-202e-4007-836f-751bd243e41a.png)

**Reply by AlbertoCuadra** [reply]
> Hey,
> 
> I tested your strategy and on changing two values, it increased
> 
> * Net Profit from **1819%** to **2206%**
> * Profit Factor from **4.482** to **8.152**
>   on 4H BTC/USDT Binance.
> 
> **Change 1: Line 73**
> 
> Before - 27 `short_RSI = input.float(27, 'Exit SRSI Short (%)', minval=0.0, step=1, group='Strategy 2: Stoch RSI')`
> 
> My Change - 21 `short_RSI = input.float(21, 'Exit SRSI Short (%)', minval=0.0, step=1, group='Strategy 2: Stoch RSI')`
> 
> **Change 2: Line 92**
> 
> Before - 88 `MA2_period_5 = input.int(88, 'MA 2', minval=1, maxval=9999, step=1, group='Strategy 5: MA CROSS')`
> 
> My Change - 85 `MA2_period_5 = input.int(85, 'MA 2', minval=1, maxval=9999, step=1, group='Strategy 5: MA CROSS')`
> 
> **Change 3 - Line 42**
> 
> Before `MAX_TP = input.int(6, 'Max number of TP', minval=1, maxval=6, group='Take Profits')`
> 
> My Change `MAX_TP = input.int(3, 'Max number of TP', minval=1, maxval=6, group='Take Profits')`
> 
> **Change 4 - Line 36**
> 
> Before `stoploss_perc= input.float(6.0, 'Stop Loss %', minval=0, maxval=100, group='Stop loss') * 0.01`
> 
> My Change `stoploss_perc= input.float(8, 'Stop Loss %', minval=0, maxval=100, group='Stop loss') * 0.01`
> 
> Let me know what do you think, my username on TradingView is ImThour. Let's have a chat, cheers.

This contribution now appears as a "Community Settings". Feel free to share your settings.

Cheers!

**Comment by 277292590** [comment]
thank you
